src/components/PupilViews/PupilQuiz/PupilQuiz.view.tsx

```diff
@@ -134,17 +139,22 @@ const QuizInner = () => {
         />
       }
       counterSlot={
-        <OakQuizCounter
-          counter={currentQuestionIndex + 1}
-          total={numQuestions}
-        />
+        !isExplanatoryText && (
+          <OakQuizCounter
+            counter={currentQuestionDisplayIndex + 1}
+            total={numInteractiveQuestions}
+          />
+        )
       }
       heading={currentSection === "starter-quiz" ? "Starter Quiz" : "Exit Quiz"}
       lessonSectionName={currentSection}
       mobileSummary={
-        <OakSpan $color={"text-primary"} $font={"body-3"}>
-          Question {currentQuestionIndex + 1} of {numQuestions}
-        </OakSpan>
+        !isExplanatoryText && (
+          <OakSpan $color={"text-primary"} $font={"body-3"}>
+            Question {currentQuestionDisplayIndex + 1} of{" "}
+            {numInteractiveQuestions}
+          </OakSpan>
+        )
```

Comment:
Looking at the designs the counter is supposed to be present for explanatory text (I guess it just doesn't advance until you're viewing the next question)

<img width="1832" alt="Screenshot 2024-02-21 at 15 33 38" src="https://github.com/oaknational/Oak-Web-Application/assets/122096/d4e1429f-fdfa-4fe3-a998-e6ae8a0c88bc">

Reply:
OK, as soon as I posted this I saw the conversation in Slack about dropping it 😆

Reply:
sorry should have added a comment to that effect